src/metrics_server/connection_metrics.spec.ts

```diff
@@ -18,6 +18,21 @@ import {
   postConnectionMetrics,
 } from './connection_metrics';
 import {InsertableTable} from './infrastructure/table';
+import {HourlyConnectionMetricsReport, HourlyUserConnectionMetricsReport} from './model';
+
+const VALID_USER_REPORT = {
+  userId: 'uid0',
+  countries: ['US', 'UK'],
```

Comment:
This is not a valid report. We only report one country per entry.
The multiple countries is legacy and not generated anymore. Don't use in new code.

Reply:
Well my point was that it's not backwards incompatible in practices because we haven't seen any servers send multiple countries. I guess I don't know how long ago sending multiple countries was still done.

I'll update the report here to have 1 country to reflect a more realistic report that we actually see.